spec/models/grantee_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe Grantee, type: :model do
+  let(:hses_grantee) { Api::FakeData::Grantee.new.data }
+  subject { described_class.new hses_grantee }
+
+  before do
+    hses_grantee[:attributes].merge!({
+    })
+  end
+
+  describe "initialize" do
+    it "sets @id" do
+      expect(subject.id).to eq hses_grantee[:id]
+    end
+
+    it "sets @attributes" do
+      expect(subject.attributes).to eq hses_grantee[:attributes].with_indifferent_access
+    end
+  end
+
+  describe "#name" do
```

Comment:
[thought] hmmm in these tests (and the `complaint_spec` which i assume these are based off) we don't really test the unhappy path. we probably should think about adding tests that indicate what we want to happen if the attribute is missing.

Reply:
adding TODO comments